### DISCUSSION TITLE
Getting error while running scripts through make file

### DISCUSSION THREAD

**Discussion by talhaEth** [Cyfrin/foundry-full-course-cu#378] [Q&A] [answered]
Hi !!!
I am getting this error while running commands using make file
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/66763322/c15b5a3c-cc66-4c8d-85b8-e5fe7c4a65e2)

here is my make file
```
-include .env

build:; forge build

deploy-sepolia:
    forge script script/DeployFundMe.s.sol:DeployFundMe --rpc-url $(SEPOLIA_RPC_URL)
	--private-key $(PRIVATE_KEY) --broadcast --verify --etherscan-api-key 
	$(ETHERSCAN_API_KEY) -vvvv


```

**Comment by talhaEth** [comment]
when I manually run this command, this command is works for me but this command is not working in make file 

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/66763322/ed459738-0b3b-416f-bd83-87582ae49d1a)

**Comment by Niferu** [comment]
Hello @talha073 

Is your `deploy-sepolia:` command in one line or you have used tabs/enters to format it to see whole command? In case you formatted it to see everything this is causing errors as commands in makefile have to be in one same line and you should only format view of it by using `alt + z`.

**Reply by talhaEth** [reply]
@Neftyr 
I used `alt + z` to format the command. but I also tried to run this command in a single line. in both cases, it did not work and throw me the same error `Makefile:6: *** multiple target patterns.  Stop.`
and one more thing, If I comment `make deploy-sepolia` command and then run the `make build` then it works

**Reply by Niferu** [reply, marked as answer]
@talha073 
So it is exactly what I have said. You just have incorrectly formatted `deploy-sepolia` command. Try copying it from course repo as you must have some additional spaces or something else in yours, which causes you this error and this should be fixed ;)

**Reply by talhaEth** [reply]
thank you @Neftyr 
It works, 
stay blessed
